ivy/functional/ivy/norms.py

```diff
@@ -28,8 +37,46 @@ def layer_norm(x, normalized_idxs, epsilon=None, scale=None, offset=None, new_st
 
     Returns
     -------
-    ret
+     ret
         The layer after applying layer normalization.
+    
+    Example
+    -------
+
+    >>> arr = ivy.full([2, 3], 13, dtype=ivy.float32)
+    >>> norm = ivy.layer_norm(arr, [0, 1], new_std=2.0)
+    >>> print(norm)
+    ivy.array([[0., 0., 0.],
+           [0., 0., 0.]], dtype=float32)
+
+    >>> arr = ivy.array([[1., 2., 3.], [4., 5., 6.]])
+    >>> norm = ivy.layer_norm(arr, [0, 1])
+    >>> print(norm)
+    ivy.array([[-1.4638476 , -0.8783086 , -0.29276952],
+           [ 0.29276952,  0.8783086 ,  1.4638476 ]], dtype=float32)
+
+    >>> arr = ivy.array([[ 0.0976, -0.3452,  1.2740], \
+        [ 0.1047,  0.5886,  1.2732], \
+        [ 0.7696, -1.7024, -2.2518]])
+    >>> norm = ivy.layer_norm(arr, [0, 1, 2], epsilon=0.001, \
+                 new_std=1.5, offset=0.5, scale=0.5)
+    >>> print(norm)
+    ivy.array([[ 0.57629204,  0.29217   ,  1.3311275 ],
+           [ 0.58084774,  0.89134157,  1.3306142 ],
+           [ 1.0074799 , -0.5786756 , -0.9311974 ]], dtype=float32)
+
+    >>> import tensorflow as tf # ivy.NativeArray
```

Comment:
You don't need to import tensorflow. You can create a new instance of native array using `ivy.native_array`
For eg.
```
With :code:`ivy.NativeArray` input:

>>> x = ivy.native_array([0, 1, 2])
>>> y = ivy.tan(x)
>>> print(y)
ivy.array([0., 1.5574077, -2.1850398])
```
Above is mentioned in the guidelines (https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html)

Reply:
Hiya 👋🏽 @bicycleman15 . Thanks for the review

Using `ivy.native_array` was my first thought. I tried it with three backends (tensorflow, torch and numpy) and they all threw No Attribute errors. That's why I defaulted to using an actual framework array to show that the function work with the native arrays of the wrapped frameworks and was agnostic

**Below are the AttributeErrors from all 3 backends:**
Tensorflow:
` arr = ivy.native_array([0.1, 2.1, 3.])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'ivy.functional.backends.tensorflow' has no attribute 'native_array'`

Torch:
`arr = ivy.native_array([0.1, 2.1, 3.])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'ivy.functional.backends.torch' has no attribute 'native_array'`

Numpy:
`arr = ivy.native_array([0.1, 2.1, 3.])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'ivy.functional.backends.numpy' has no attribute 'native_array'`


I hope that clears things up on my reasons. Thanks

Reply:
Ah okay.

Maybe for now, let's ignore this error. It is something that we will have to fix. 🙂

Reply:
I would still say make it `ivy.native_array` only. Don't worry about the failing test.